templates/navbar.html

```diff
@@ -1,8 +1,9 @@
-<nav class="navbar navbar-toggleable-md navbar-inverse fixed-top bg-inverse">
-     <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+<nav class="navbar navbar-expand-lg navbar-inverse fixed-top navbar-dark bg-dark">
```

Comment:
Can you explain the changes?

Reply:
Hi @tulikavijay, 
The responsive navbar toggle has been updated in bootstrap 4, we can specify the background color using bg- classes: `bg-light`, `bg-dark`, etc. and for _lighter_ text color we can use `navbar-dark`.